VideoLocker/src/main/java/org/edx/mobile/task/GetResponsesListTask.java

```diff
@@ -26,8 +26,12 @@ public ThreadComments call() throws Exception {
         try {
             if (threadId != null) {
                 if (isQuestionType) {
+                    if (shouldGetEndorsed) {
+                        return environment.getDiscussionAPI().getResponsesListForQuestion(threadId,
```

Comment:
I added comments in this section, and also edited the javadoc of the `getEndorsedList()` method in  `ResponsesLoader`.